walnut/converter/converter.go

```diff
@@ -111,13 +85,58 @@ func ExtractTaskClasses(dplDump Dump) (tasks []*task.Class, err error) {
 				Memory: createFloat(128),
 				Ports:  task.Ranges{}, //begin - end OR range
 			},
-			Bind: bind,
 			Properties: gera.MakeStringMapWithMap(map[string]string{
 				"severity": "trace",
 				"color":    "false",
 			}),
-			Connect: connect,
 		}
+
+		module = append(module, "Control-OCCPlugin")
+		loadModule := "eval `aliswmod load "
```

Comment:
I advise using `fmt.Sprintf` and `strings.Join` for more idiomatic string building.

Reply:
Excellent suggestion. This converted my 10 line implementation into a single line while also getting rid of the `if` statement.